builder.go

```diff
@@ -580,6 +582,15 @@ func ParseDockerignore(root string) ([]string, error) {
 	return excludes, nil
 }
 
+// ParseDockerIgnore returns a list of the excludes in the .containerfile or .dockerignore file.
+func ParseDockerignore(root string) ([]string, error) {
+	excludes, err := ParseIgnore(filepath.Join(root, ".containerignore"))
+	if err != nil && !os.IsNotExist(err) {
```

Comment:
This condition looks like it should be `if err != nil && os.IsNotExist(err)`.

Reply:
Fixed